### DISCUSSION TITLE
BORG_PASSCOMMAND not working, unable to add repo to Vorta

### DISCUSSION THREAD

**Discussion by the-solipsist** [borgbackup/borg#8097] [Q&A]
I use `pass` as a password manager.  I recently thought I would use the `BORG_PASSCOMMAND` env variable along with `pass`.  Accordingly, I did `set -Ux BORG_PASSCOMMAND 'pass show borg_backup'` (using the `fish` shell).

After I did that, my Vorta backups started failing:
```
"ERROR: passcommand supplied in BORG_PASSCOMMAND failed: Command '['pass', 'show', 'borg_backup']' returned non-zero exit status 1.
```

I tried erasing the `BORG_PASSCOMMAND` env variable, but that didn't help.

I unliked the repo, and tried re-adding it using a passphrase, but I still get the same error, and I'm unable to even add my repo to Vorta.

Does anyone have any idea how to fix this?

**Comment by infectormp** [comment]
export BORG_PASSCOMMAND='pass show borg_backup' ?

**Reply by the-solipsist** [reply]
That's the equivalent command for bash shell. I tried that too. It doesn't help. 

UPDATE: fat finger mistake. It should be `pass show borg-backup` not `pass show borg_backup`. Sorry!
